package.json

```diff
@@ -16,22 +16,28 @@
   },
   "scripts": {
     "start": "node index.js",
-    "lint": "eslint --cache ./cloud && eslint --cache index.js",
-    "lint-fix": "eslint --cache --fix ./cloud && eslint --cache --fix index.js",
-    "test": "jasmine"
+    "lint": "eslint --cache ./cloud && eslint --cache index.js && eslint --cache ./spec",
+    "lint-fix": "eslint --cache --fix ./cloud && eslint --cache --fix index.js && eslint --cache  --fix./spec",
+    "test": "jasmine",
+    "coverage": "nyc jasmine",
+    "prettier": "prettier --write '{cloud,spec}/{**/*,*}.js' 'index.js'",
+    "watch": "babel-watch index.js"
   },
   "engines": {
     "node": ">=4.3"
   },
   "devDependencies": {
+    "babel-eslint": "^10.1.0",
```

Comment:
Not sure whether we should keep ranges for dev or any dependency. This example repo demonstrates the features of the dev dependencies, so maybe they should be locked in?